test/Dapr.Client.Test/InvokeApiTest.cs

```diff
@@ -381,6 +382,42 @@ public async Task InvokeMethodAsync_WithReturnTypeAndData_WithNonDefaultVerb_Wit
             response.Name.Should().Be(invokedResponse.Name);
         }
 
+        [Fact]
+        public async Task InvokeMethodAsync_AppCallback_SayHello()
+        {
+            // Configure Client
+            var jsonOptions = new JsonSerializerOptions { PropertyNamingPolicy = JsonNamingPolicy.CamelCase };
+            var httpClient = new AppCallbackClient(new DaprAppCallbackService(jsonOptions));
+            var daprClient = new DaprClientBuilder()
+                .UseGrpcChannelOptions(new GrpcChannelOptions { HttpClient = httpClient })
+                .UseJsonSerializationOptions(jsonOptions)
+                .Build();
+
+            var request = new Request() { RequestParameter = "John" };
```

Comment:
lets remove specific names from the code.

Reply:
Done